configuration/pih/petl/jobs/sql/extractions/pmtct_infant_delivery.sql

```diff
@@ -0,0 +1,123 @@
+set sql_safe_updates = 0;
+set session group_concat_max_len = 100000;
+
+select program_id into @eidProgram from program where uuid = '7e06bf82-9f1a-4218-b68f-823082ef519b';
+select encounter_type_id into @labResultsEnc from encounter_type where uuid = '4d77916a-0620-11e5-a6c0-1697f925ec7b';
+select name into @hivDispEncName from encounter_type where uuid = 'cc1720c9-3e4c-4fa8-a7ec-40eeaad1958c';
+
```

Comment:
Assuming you've double checked the uuids above.  You can use variables here like ${encounterType.HIV_DISPENSING.uuid}, but obviously that makes the SQL harder to test in a SQL editor